exporters/holstebro_helpers.py

```diff
@@ -0,0 +1,811 @@
+                    node.name, 'engagement', False)
+                for uuid, employee in employees.items():
+                    row = {}
+                    address = mh.read_user_address(uuid, username=True, cpr=True)
+
+                    # is this employee the manager for the department? Then fetch the parent ou's manager
+                    if uuid != manager['uuid']:
+                        responsible = manager_engagement[0]['user_key']
+                    else:
+                        parent_manager = find_org_manager(mh, node.parent)
+                        if(parent_manager['uuid'] != ''):
+                            pmanager_engagement = mh.read_user_engagement(
+                                parent_manager['uuid'])
+                        else:
+                            pmanager_engagement = [{'user_key': ''}]
+
+                        responsible = pmanager_engagement[0]['user_key']
+
+                    row = {'integrationsid': uuid,
+                           'Username': f"HK-{employee['User Key']}",
+                           'Name': employee['Navn'],
+                           'Title': employee['Titel'] if employee['Titel'] is not None else employee['Stillingsbetegnelse'],
+                           'Address': address['Lokation'] if 'Lokation' in address else '',
+                           'Email': address['E-mail'] if 'E-mail' in address else '',
+                           'Mobile': address['Mobiltelefon'] if 'Mobiltelefon' in address else '',
+                           'Telephone': address['Telefon'] if 'Telefon' in address else '',
+                           'Responsible': f"HK-{responsible}" if responsible != '' else '',
+                           'Company': employee['Org-enhed UUID']
+                           }
+
+                    rows_persons.append(row)
+
+    logger.info(f"Writing {len(rows_persons)} rows to {filename_persons}")
+    mh._write_csv(fieldnames_persons, rows_persons, filename_persons)
+
+    logger.info(f"Writing {len(rows_org)} rows to {filename_org}")
+    mh._write_csv(fieldnames_org, rows_org, filename_org)
+
+
+def update_org_with_hk_managers(mh, nodes):
+    """ Traverses a tree of OUs, for each OU finds the manager of the OU.
+    : param nodes: The nodes of the OU tree
+    """
+    managerHelper = HolstebroHelper(mh)
+
+    for node in PreOrderIter(nodes['root']):
+        ou = mh.read_ou(node.name)
+        # for each ou, check if name contains _leder
+        # if so, check ou for "tilknytninger" and set this as leader for ou.parent
+        if _is_leader_unit(ou):
+            # We have an Afdeling with name _leder, find associated employees and make them leaders in the parent ou
+            associated_employees = mh.read_organisation_people(
+                node.name, 'association', False)
+
+            # Find the manager if an employee is associated with manager departement
+            manager = {}
+            if len(associated_employees) > 0:
```

Comment:
Dette er ækvivalent:
```suggestion
            if associated_employees:
```